packages/connect-web/package.json

```diff
@@ -35,8 +35,8 @@
         "build:lib": "yarn g:rimraf ./lib && yarn g:tsc --build tsconfig.lib.json && ../../scripts/replace-imports.sh ./lib",
         "dev": "yarn g:rimraf build && TS_NODE_PROJECT=\"tsconfig.lib.json\" yarn webpack --config ./webpack/dev.webpack.config.ts",
         "build:inline": "TS_NODE_PROJECT=\"tsconfig.lib.json\" webpack --config ./webpack/inline.webpack.config.ts",
-        "build:webextension": "TS_NODE_PROJECT=\"tsconfig.lib.json\" yarn webpack --config ./webpack/prod.webpack.config.ts",
-        "build": "yarn g:rimraf build && yarn build:inline && yarn build:webextension",
+        "build:webextension": "TS_NODE_PROJECT=\"tsconfig.lib.json\" webpack --config ./webpack/prod.webpack.config.ts",
+        "build": "rm -rf build && yarn build:inline && yarn build:webextension",
```

Comment:
Isn't this gonna break the build on win? Why `rm -rf` here and `yarn rimraf` in connect-iframe?

Reply:
According to @vdovhanych it should be fine, also we agreed to remove rimraf completely because everyone uses Mac/Linux or with Widnows every person uses WSL which can do `rm` -rf too.

Reply:
it shouldnt break the build for us as its either on mac or on Linux using the wine builder

Reply:
I don't think it would be a problem for any of us, but it might be for some 3rd party developer or someone who wants to build it themselves... So I would prefer rimraf because it seems to me that it costs next to nothing. But if it has some drawbacks or costs that I don't see, then of course I'm not opposed to switching to `rm -rf` to make it work especially for us.

Reply:
I'm with @Nodonisko on this. Any sane person should be using WSL. Also, the build on Windows is broken anyway; that's also the reason why I ended up building desktop windows app on Linux with wine.